IntrinsicCategories/gap/IntrinsicCategories.gi

```diff
@@ -678,14 +678,12 @@ InstallMethod( IntrinsifyObject,
   function( C, o )
     local obj;
     
-    obj := rec(
-               PositionOfLastStoredCell := 1,
-               PositionOfActiveCell := 1,
-               TransitionIsomorphisms := rec( ),
-               1 := o
-               );
+    obj := CreateCapCategoryObjectWithAttributes( C );
```

Comment:
I just looked at the CI failure: The problem is that this line triggers an immediate method which calls code expecting `PositionOfActiveCell` to be set:
https://github.com/homalg-project/homalg_project/blob/71a6b6d1b8a0930fb50ffb40d59a200b36558a3c/Modules/gap/LIMOD.gi#L751

Since this seems to be rooted deeply in homalg, I fear there will not be an easy solution. @mohamed-barakat Do you have an idea?

Reply:
I would suggest taking any code related to `Intrinsic*` out of this PR.

Reply:
Alright, done.